torch/onnx/symbolic_opset10.py

```diff
@@ -0,0 +1,25 @@
+import torch
+import torch.onnx
+# This import monkey-patches graph manipulation methods on Graph, used for the
+# ONNX symbolics
+import torch.onnx.utils
+
+from torch.onnx.symbolic_helper import parse_args, _unimplemented
+import torch.onnx.symbolic_opset9
+
```

Comment:
Add some comment for the opset detaisl, such as when the opset is supported by which onnx release version, and release date and so on.